pravega-k8-log-collection/pravega-k8s-fluentbit-log.sh

```diff
@@ -0,0 +1,94 @@
+#/usr/bin/env bash
+#
+# Pravega K8 based log collection tool
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+set -euo pipefail
+# Setting up temporary log collection directory under /tmp/logdir
+mkdir -p ./logdir
+output_dir="./logdir"
```

Comment:
These commands can be inverted and make `mkdir` use the `output_dir` variable.

Reply:
Done.